components/hab/src/command/studio/enter.rs

```diff
@@ -43,6 +44,29 @@ fn set_env_var_from_config(env_var: &str, config_val: Option<String>, sensitive:
     }
 }
 
+fn cache_ssl_cert_file(cert_file: &str, cert_cache_dir: &Path) {
+    let cert_path = Path::new(&cert_file);
+
+    if cert_path.exists() && cert_path.is_file() {
+        let cert_filename = match cert_path.file_name() {
+            Some(name) => name,
+            None => {
+                debug!("Unable to determine file name for SSL_CERT_FILE={}",
+                       &cert_file);
+                return;
+            }
+        };
+        let cache_file = cert_cache_dir.join(&cert_filename);
+        debug!("Caching SSL_CERT_FILE {}", &cert_file);
+
+        if let Err(err) = stdfs::copy(cert_file, &cache_file) {
+            debug!("Unable to cache SSL_CERT_FILE: {}", err);
+        }
+    } else {
+        debug!("Invalid path for SSL_CERT_FILE: {}", &cert_file);
```

Comment:
I think this should be a warn or error too and I'd also add something like "check to make sure that file exists"

Reply:
@mwrock @baumanj Do you think it is worth passing the `UI` into this function so we can emit this information as part of the normal output, or is `warn!` sufficient?

Reply:
You could pass the `UI` which I think is better than `warn!` but perhaps even better whould be to have `cache_ssl_cert_file` return a `Result` and then `start` deals with the err using its `UI`.

Reply:
Note there is a separate card to plumb in the cert status into the UI so might be better to do that work together instead of in this PR